### ISSUE TITLE
[DEV]: Replace `pl.to_series()[0]` with `.row(by_predicate="...")`

### ISSUE THREAD

**Issue by jjfantini** [humblFINANCE/humblDATA#8]
Instead of converting objects to a series and then pulling the value from DataFrames that have `data.shape == (1,1)`, just use `.row()` to extract the values easily

**Comment by jjfantini** [comment]
```py
import polars as pl
import numpy as np

# Create a simple DataFrame
df = pl.DataFrame(
    {
        "A": np.random.rand(10000),
        "B": np.random.rand(10000),
        "C": np.random.rand(10000),
    }
)


# Using .to_series()[0]
%timeit df.to_series()[0]
# Using .row(0)
%timeit df.row(0)[0]


1.48 µs ± 16.7 ns per loop (mean ± std. dev. of 7 runs, 1,000,000 loops each)
572 ns ± 5.89 ns per loop (mean ± std. dev. of 7 runs, 1,000,000 loops each)
```

### Performance Boost ⚡: 2.54x faster (154%)

**Comment by jjfantini** [comment]
closed with commit: `affad54`
